WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -970,7 +963,7 @@ protected Address doInBackground(Double... args) {
         }
 
         protected void onPostExecute(@Nullable Address address) {
-            if (!isAdded() || address == null || address.getMaxAddressLineIndex() == 0) {
+            if (address == null || address.getMaxAddressLineIndex() == 0) {
```

Comment:
> after chatting with @maxme about the whole thing, if I didn't misunderstand him, the only time we actually need to check for isAdded is before accessing getActivity since that can be null. Am I wrong about this?

I believe that's *mostly* correct. The tricky part is some things internal to `Fragment.java` rely on the activity without you knowing it. For example, calling `getResources()` in a fragment internally calls `activity.getResources()` and will will throw an exception if the activity doesn't exist. I recommend viewing the source of `Fragment.java` and searching for `if (mHost == null`.